build.ps1

```diff
@@ -323,12 +323,19 @@ if (($All -or $BuildJava) -and -not $NoBuildJava) {
     }
     else {
         try {
-            $jdkVersion = (Get-Item HKLM:\SOFTWARE\JavaSoft\JDK | Get-ItemProperty -name CurrentVersion).CurrentVersion
-            $javaHome = (Get-Item HKLM:\SOFTWARE\JavaSoft\JDK\$jdkVersion | Get-ItemProperty -Name JavaHome).JavaHome
-            if (Test-Path "${env:JAVA_HOME}\bin\java.exe") {
```

Comment:
☝️ This was the bug

Reply:
Thanks, I noticed the same problem this morning and merged the same fix in a different PR.